### ISSUE TITLE
Horizontal pager display issue

### ISSUE THREAD

**Issue by JulianKingman** [shoutem/ui#297]
The horizontal pager has an odd display glitch that appears only on initial load, and then disappears.  Adding `overflow: 'hidden'` to the pager masks the problem, but it still doesn't slide correctly on first load. I'll look through the code and see if anything jumps out at me as the source of the problem.

![horizontalpagerbug](https://user-images.githubusercontent.com/1566400/28896835-0f4ceb9c-77ac-11e7-9774-605377ffedc1.gif)

**Comment by JulianKingman** [comment]
A simple check for `containerWidth` in `renderPages` fixes it, for example:
```javascript
const pageContent = this.shouldRenderPage(pageIndex) && (
        <Page
          isActive={isPageActive}
          width={pageWidth}
          style={style.page}
        >
          {renderPage(pageData, pageIndex, { width, height })}
        </Page>
      );

      if (!containerWidth) return null;

      return (
        <View key={pageIndex} style={{ width: containerWidth }}>
          {pageContent}
        </View>
      );
    });
```
It's such a little thing it seems like you could just put it in, but if you want I'll make a PR.

**Comment by Vladimir-Vdovic** [comment]
A pull request would be really appreciated, the dev team can review it and get it merged quicker.

**Comment by JulianKingman** [comment]
I submitted a PR

**Comment by zrumenjak** [comment]
The fix from #298 has been released in v0.21.1
